src/registryctl/config/registry/config.go

```diff
@@ -0,0 +1,28 @@
+package registry
+
+import (
+	"fmt"
+	"github.com/docker/distribution/configuration"
+	storagedriver "github.com/docker/distribution/registry/storage/driver"
+	"os"
+)
+
+// StorageDriver ...
+var StorageDriver storagedriver.StorageDriver
```

Comment:
Why not make storagedriver part of the config of registryctl?

Reply:
The configuraitons of the driver are arleady in the registry configuration file, and the driver is inited on registryctl container launch. I think it's ok to use a varaible here.

Reply:
I strongly think putting in in config is a better design.
I'll let @heww chime in ,if he's OK with this I'm fine

Reply:
If putting `StorageDriver` in the config of the registryctl, it will be clear in the main of the registryctl. 

In the `Load` method of the registryctl conf, we can resolve configuration of the distrubtion and create `StorageDriver` from the configurtion. The result is that is subpackage `config/registry` can be removed.

Reply:
added.